openlibrary/solr/update_work.py

```diff
@@ -739,53 +743,65 @@ def add(name, value):
         def add_list(name, values):
             doc[name] = list(values)
 
+        borrowable_editions = set()
+        printdisabled_editions = set()
+        open_editions = set()
+        unclassified_editions = set()
+
+        language_editions = {}
+
         pub_goog = set()  # google
         pub_nongoog = set()
         nonpub_goog = set()
         nonpub_nongoog = set()
```

Comment:
A lot of these are no longer used; let's axe them. They'll appear greyed-out in vs code when unused.